Assets/Scripts/MVC/EnemyTank/EnemyTankView.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.AI;
+
+public class EnemyTankView : MonoBehaviour
+{
+    public GameObject Turret;
```

Comment:
All elements are public here why we are exposing all elements public .
If its class specific then make it private and if we need this in inspector panel then make it serialized and private and if you need it outside  the class make it internal or protected .